specs/windows/bios_info.table

```diff
@@ -0,0 +1,10 @@
+table_name("bios_info")
+description("Lists important information from the system bios.")
+schema([
+  Column("Name", TEXT, "AMT Status"),
```

Comment:
These descriptions don’t make much sense. Perhaps Name would be `BIOS setting name` and Value would be `Value if the bios setting`? Feel free to play around with other things, but these should better describe what data is being given back.